### PR TITLE
Updated button styling and preview images

### PR DESCRIPTION
**Pull request by egrace479**
Updated the styling of the buttons within the dashboard to match the `Upload` button and replaced the preview images with new images including the upload button (was not previously an option).

Also fixed the `Image_filename`/`file_url` optional requirement description in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 Prototype data dashboard using the [Cuthill Gold Standard Dataset](https://huggingface.co/datasets/imageomics/Curated_GoldStandard_Hoyal_Cuthill), which was processed from Cuthill, et. al. (original dataset available at [doi:10.5061/dryad.2hp1978](https://doi.org/10.5061/dryad.2hp1978)). Test datasets (the processed version of Cuthill's data with and without filepath URLs) are available in [test_data](./test_data).
 
 
-### How it works
+## How it works
 
 For full dashboard functionality, upload a CSV or XLS file with the following columns: 
-- `Image_filename`: Filename of each image, must be unique. **Note:** These should be `png` or `jpg`, `tif` may not show image.
+- `Image_filename`*: Filename of each image, must be unique. **Note:** These should be `png` or `jpg`, `tif` may not show image.
```
Comment (hlapp) [inline README.md:8]:
```suggestion
- `Image_filename`*: Filename of each image, must be unique. **Note:** Images should be in PNG or JPEG format, TIFF may fail to render in the sample image display.
```

File: README.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ For full dashboard functionality, upload a CSV or XLS file with the following co
 
 ***Note:** 
 - `lat` and `lon` columns are not required to utilize the dashboard, but there will be no map view if they are not included.
-- `file_urls` is not required, but there will be no sample images option if it is not included.
+- `Image_filename` and `file_url` are not required, but there will be no sample images option if it they not both included.
```
Comment (hlapp) [inline README.md:20]: It's a little unclear to me whether you mean to say _if neither is included_, or _if either one is not included_.

**Review by hlapp — APPROVED** [review]
See inline suggestion and comment.

File: README.md (hunk 2 above)
Comment (egrace479) [reply README.md:20]: It was supposed to be "if they are not both included", though I like the suggested "if either one is not included"

**Review by thompsonmj — APPROVED** [review]

**Pull request merged**